mucommander-core/src/main/java/com/mucommander/ui/dialog/about/AboutDialog.java

```diff
@@ -189,6 +189,7 @@ private JScrollPane createCreditsPanel() {
             insertDetailedUrl(doc,    "Yanfs",               "BSD",                                  "http://yanfs.dev.java.net");
             insertDetailedUrl(doc,    "JCommander",          "Apache License",                       "http://jcommander.org");
             insertDetailedUrl(doc,    "ICEpdf",              "Apache License",                       "http://www.icesoft.com/icepdf");
+            insertDetailedUrl(doc,    "RSyntaxTextArea",     "BSD 3-Clause New or Revised License",  "https://github.com/bobbylight/RSyntaxTextArea");

```

Comment:
the link should rather be to their website
```suggestion
            insertDetailedUrl(doc,    "RSyntaxTextArea",     "BSD 3-Clause New or Revised License",  "https://bobbylight.github.io/RSyntaxTextArea/");

```